src/main/java/org/codeforamerica/shiba/Utils.java

```diff
@@ -10,6 +10,10 @@
 
 @Slf4j
 public class Utils {
+    private Utils() {
+        throw new AssertionError("Cannot instantiate utility class");
+    }
```

Comment:
This is to encourage static references and prevent instantiation right?

Reply:
Yeah exactly. I realized our utility classes were instantiable and decided to add this guard just to make sure we don't enable future devs to do something weird.